live-examples/css-examples/fragmentation/break-before.html

```diff
@@ -0,0 +1,36 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="break-before">
+    <div class="example-choice">
+        <pre><code class="language-css">break-before: auto;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">break-before: page;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">break-before: left;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
```

Comment:
I am afraid that values `left` and `right` are not really working on Firefox or Chrome: https://caniuse.com/css-page-break
Should I remove them to avoid confusion?
I also don't see any way to affect the default value of `Pages per sheet`.

Reply:
We can keep the example if you think it will not cause confusion or make people wonder about the outputs. Otherwise, let's add these examples later when we're able to better demonstrate them.

Reply:
I think it's better to keep this example. I removed `left` values, so there will be no confusion.